.github/workflows/image-check.yaml

```diff
@@ -12,6 +12,8 @@ concurrency:
 jobs:
   diff:
     runs-on: distroless-ci-large-ubuntu-20.04 # custom runner most compatible with debian 11
+    permissions:
+      pull-requests: write
```

Comment:
I added this as the workflow only gets `read` permissions if the author of a PR is outside the organisation. But the `peter-evans/create-or-update-comment` action needs `write` access to be able to update the PR.

Reply:
I don't think we should expand permissions. Don't worry about the diff image check.

Reply:
reverted